tests/Dockerfile

```diff
@@ -7,8 +7,11 @@ RUN apt-get update && apt-get install -y \
     software-properties-common \
     python-pip
 
+RUN apt-add-repository --yes --update ppa:ansible/ansible && \
+    apt-get -y install ansible
+
 RUN pip install -U pip
```

Comment:
```suggestion
RUN pip install pip==20.1.1
```

Reply:
20.2 is busted on Python 2. Pin 20.1.1 and there's no need for the ppa. Let's keep the install method as-is.

While we're here Python 2 has been EoL since Jan, and this doesn't affect Python 3.

Reply:
@BradleyBoutcher let's file a follow-up issue to upgrade the test flow to use Python 3

Reply:
Issue created: https://github.com/cyberark/ansible-conjur-host-identity/issues/41
Solution suggested by Gary has been implemented in this PR as well